test_utils/venv.py

```diff
@@ -18,6 +18,8 @@ def run_in_virtualenv(shell_script):
         venv_path = str(Path(parent_dir) / 'venv')
         try:
             subprocess.check_call(['virtualenv', '-p', sys.executable, '--clear', venv_path])
+            # TODO: Remove the following line when both the latest versions of pip and pip-tools work fine together
```

Comment:
Not sure we need the `TODO` comment; wouldn't we just want to always use the pip version the repo is currently using in `pip.txt`?

Reply:
yeah, I think we don't need this comment